res/values/strings.xml

```diff
@@ -228,11 +228,11 @@
     <string name="jump_to_message">Jump to message</string>
     <string name="copy_json">Copy JSON</string>
     <string name="replace_draft">Replace draft</string>
-    <string name="title_share_location">Share location with all group members</string>
+    <string name="title_share_location">Share your location with the group</string>
```

Comment:
"the group" seems to be less specific. but i am wondering if there are issues with the existing string at all.

Reply:
yes, because it isn't "all group members excluding group owners".